app/controllers/employees_controller.rb

```diff
@@ -0,0 +1,52 @@
+class EmployeesController < ApplicationController
+
+  def register_employee
+  end
+
+  def index
+    @employees = Employee.all
+  end
+
+  def show
+    @employee = Employee.find(params[:id])
+  end
+
+  def new
+    @employee = Employee.new
+  end
+
+  def create
+    @employee = Employee.new(employee_params)
+    if @employee.save
+      redirect_to "/employees"
+    else
+      render :new, status: :unprocessable_entity
+    end
+  end
+
+  def edit
+    @employee = Employee.find(params[:id])
+  end
+
+  def update
+    @employee = Employee.find(params[:id])
+
+    if @employee.update(employee_params)
+      redirect_to '/employees'
+    else
+      render :edit, status: :unprocessable_entity
+    end
+  end
+
+  def destroy
+    @employee = Employee.find(params[:id])
```

Comment:
You can refactor by doing something like this 
`def set_employee
      @employee = Employee.find(params[:id])
   end`

What do you think